python-sdk/example_dags/example_load_file.py

```diff
@@ -19,6 +19,7 @@
     "retries": 1,
     "retry_delay": 0,
 }
+data_url = "https://archive.ics.uci.edu/ml/machine-learning-databases/iris/iris.data"  # URL for Iris data API
```

Comment:
@sunank200 Let's not add test cases dependency to external URL, can we move this to S3/GCS?